### PR TITLE
✨ Added Vidazoo Widget Extension

### PR DESCRIPTION
**Pull request by uditalias**
Implementation of the Vidazoo Widget AMP extension

Usage:

```html
<amp-vidazoo-widget
    data-widget-id="5fe8ba17de03c70004db1d48"
    layout="responsive"
    width="16"
    height="9"></amp-vidazoo-widget>
```

I2I Issue: [31884](https://github.com/ampproject/amphtml/issues/31884)

### CODE REVIEW

File: extensions/amp-vidazoo-widget/0.1/test/validator-amp-vidazoo-widget.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  Tests for the amp-vidazoo-widget tag.
+-->
+<!doctype html>
+<html ⚡>
+
+<head>
+    <meta charset="utf-8">
+    <link rel="canonical" href="./regular-html-version.html">
+    <meta name="viewport" content="width=device-width">
+    <style amp-boilerplate>
```
Comment (twifkak) [inline extensions/amp-vidazoo-widget/0.1/test/validator-amp-vidazoo-widget.html:24]: For succinctness, can you squeeze all the amp-boilerplate CSS onto one line, as it is done at https://validator.amp.dev/ and in other validator tests? It obscures the useful part of this test, which is in the `<body>`.

File: extensions/amp-vidazoo-widget/0.1/test/validator-amp-vidazoo-widget.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+</head>
+
+<body>
+    <h3>Vidazoo Widget Responsive Layout</h3>
+    <amp-vidazoo-widget data-widget-id="5fe8ba17de03c70004db1d48" layout="responsive" width="16" height="9">
+    </amp-vidazoo-widget>
+
+    <h3>Vidazoo Widget Fixed Layout</h3>
+    <amp-vidazoo-widget data-widget-id="5fe8ba17de03c70004db1d48" layout="fixed" width="300" height="250">
+    </amp-vidazoo-widget>
```
Comment (twifkak) [inline extensions/amp-vidazoo-widget/0.1/test/validator-amp-vidazoo-widget.html:103]: Probably worth adding a test with a missing `data-widget-id` to ensure an error shows up.

**Review by twifkak — CHANGES_REQUESTED** [review]
Thanks! Reviewing validator changes only, I have a couple small requests. Validator changes look good otherwise.

File: validator/js/nodejs/package-lock.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "amphtml-validator",
-  "version": "1.0.33",
+  "version": "1.0.34",
```
Comment (twifkak) [inline validator/js/nodejs/package-lock.json:3]: Is this version change necessary to get the PR checks to pass? If not, then best to separate it into a different PR.

**Review by twifkak — APPROVED** [review]
Validator changes LGTM with one optional comment, on a file I'd missed last time. Thanks!

File: validator/js/nodejs/package-lock.json (hunk 3 above)
Comment (uditalias) [reply validator/js/nodejs/package-lock.json:3]: @twifkak Not at all, it just got updated. I'll revert it.

**Comment by uditalias** [comment]
Hey @alanorozco 👋,
Please let me know if there is anything else I should take care of.

Thanks 😊

**Comment by alanorozco** [comment]
@mfainshtein 
@uditalias 
Will this ever go to Design Review, like [I asked](https://github.com/ampproject/amphtml/issues/31884#issuecomment-842383566.)?

File: extensions/amp-vidazoo-widget/0.1/test/test-amp-vidazoo-widget.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
```
Comment (erwinmombay) [inline extensions/amp-vidazoo-widget/0.1/test/test-amp-vidazoo-widget.js:2]: 2021 on here and all newer files
